tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,63 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file_no_unexpected_comma = """
+module UnexpectedComma (..) where
+
```

Comment:
can you avoid (all|excessive|consecutive) blank lines in these sample files; it makes it hard to read, *especially* if syntax highlighting isnt present.